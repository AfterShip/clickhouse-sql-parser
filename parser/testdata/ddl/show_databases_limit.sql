SHOW DATABASES LIMIT 10