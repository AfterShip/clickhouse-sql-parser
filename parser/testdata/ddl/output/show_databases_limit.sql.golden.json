[
  {
    "ShowPos": 0,
    "StatementEnd": 24,
    "ShowType": "DATABASES",
    "Target": null,
    "NotLike": false,
    "LikeType": "",
    "LikePattern": null,
    "Limit": {
      "NumPos": 21,
      "NumEnd": 23,
      "Literal": "10",
      "Base": 10
    },
    "OutFile": null,
    "Format": null
  }
]